frontend/src/metabase/embedding/embedding-iframe-sdk-setup/components/tests/SdkIframeEmbedSetup-embedding-hub.unit.spec.tsx

```diff
@@ -1,10 +1,19 @@
 import userEvent from "@testing-library/user-event";
 
 import { screen } from "__support__/ui";
+import { PLUGIN_EMBEDDING_IFRAME_SDK_SETUP } from "metabase/plugins";
 
 import { setup, waitForUpdateSetting } from "./test-setup";
 
 describe("Embed flow > embedding hub step completion tracking", () => {
+  beforeEach(() => {
+    PLUGIN_EMBEDDING_IFRAME_SDK_SETUP.isFeatureEnabled = jest.fn(() => true);
+  });
+
+  afterEach(() => {
+    PLUGIN_EMBEDDING_IFRAME_SDK_SETUP.isFeatureEnabled = () => false;
+  });
```

Comment:
nit: we could replace this with `setupEnterpriseOnlyPlugin("embedding-iframe-sdk-setup")` which does the same thing but more idiomatic